LICENSE.3rdparty

```diff
@@ -0,0 +1,76 @@
+Files: *
+Copyright: (c) 2014, Tenzir GmbH
+License: BSD-3-clause
+
+Files: libvast/aux/robin-map/*
+Copyright: (c) 2017 Tessil
+License: MIT
+
+Files: libvast/src/concept/hashable/crc.cpp
+Copyright: (C) 1995-2013 Jean-loup Gailly and Mark Adler
+License: Zlib
+
+Files: libvast/vast/byte.hpp libvast/vast/detail/narrow.hpp libvast/vast/span.hpp
+Copyright: (c) 2015 Microsoft Corporation
+License: MIT
+Comment: From the GSL library.
+
+Files: libvast/vast/detail/math.hpp
+Copyright: (c) 2015 Orson Peters
+License: Zlib
+
+Files: libvast/vast/detail/radix_tree.hpp
+Copyright: (c) 2012, Armon Dadgar
+License: BSD-3-clause
+Comment: C++ flavor of Adaptive Radix Tree, derived from original C
+ implementation by Matthias Vallentin.
```

Comment:
This file does not exist.

Reply:
Ah, this was based off of the Debian package provided by Sascha with a few adaptions. I'll remove this one.